bootstrap/roles/appliance-build.bootstrap/tasks/main.yml

```diff
@@ -19,31 +19,94 @@
   register: toplevel
 
 - apt:
-    name: aptitude
+    name: software-properties-common
     state: present
+    update_cache: yes
 
-- apt:
+- apt_repository:
+    repo: ppa:ansible/ansible
     update_cache: yes
 
 - apt:
-    upgrade: yes
+    upgrade: dist
```

Comment:
Could you elaborate on this?

Reply:
I opted to change this to "dist" so we can drop the need for the aptitude package to be installed. I also think performing "apt-get dist-upgrade" (after) vs. "aptitude safe-upgrade" (before) is "more correct".